app/controllers/people_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PeopleController < ApplicationController
+  before_action :authenticate_user!
   before_action :set_person, only: [:show, :edit, :update, :destroy]
 
   def index
```

Comment:
should this have `authorize @people`?

Reply:
Maybe -- I'm still learning how to setup Pundit

Reply:
*shrug* Maybe? We don't really have a lot of info about the expectations of how this stuff works so I wasn't sure how the `/people` route was intended to be used. Should it be locked down?

Reply:
IME, authorizing a collection isn't very useful, but verifying that the collection was generated using a policy scope is. The best way to do that is to let the `verify_policy_scoped` after_action catch it if you forget.